packaging/nntrainer.spec

```diff
@@ -196,15 +240,32 @@ cp -r result %{buildroot}%{_datadir}/nntrainer/unittest/
 %{_includedir}/nntrainer/tensor.h
 %{_includedir}/nntrainer/lazy_tensor.h
 %{_includedir}/nntrainer/tensor_dim.h
-%{_includedir}/nntrainer/nntrainer.h
 %{_includedir}/nntrainer/nntrainer_log.h
 %{_includedir}/nntrainer/nntrainer_logger.h
 %{_includedir}/nntrainer/optimizer.h
 %{_includedir}/nntrainer/util_func.h
 %{_includedir}/nntrainer/parse_util.h
-%{_libdir}/*.a
 %{_libdir}/pkgconfig/nntrainer.pc
 
+%files devel-static
+%{_libdir}/*.a
+%exclude %{_libdir}/libcapi*.a
+
+%if %{with tizen}
+%files -n capi-nntrainer
+%manifest capi-nntrainer.manifest
+%license LICENSE
+%{_libdir}/libcapi-nntrainer.so
+
+%files -n capi-nntrainer-devel
+%{_includedir}/nntrainer/nntrainer.h
+%{_libdir}/pkgconfig/capi-nntrainer.pc
+
+%files -n capi-nntrainer-devel-static
```

Comment:
```suggestion
%files -n capi-nntrainer-devel-static
%{_includedir}/nntrainer/nntrainer.h 
```

Reply:
As long as capi-nntrainer-devel-static requires capi-nntrainer-devel, it does not need to have it and having a same file in two sub-packakges is dangerous (I personally prohibit it)

Reply:
I see. Thanks for the explanation :)